### DISCUSSION TITLE
Make Peeky and Vitest compatible and work together

### DISCUSSION THREAD

**Discussion by edimitchel** [vitest-dev/vitest#48] [Ideas]
As many people on twitter ask about @Akryum's peeky project and the new vitest project if peeky and Vitest will on day merge effort to provide one test solution with UI as option.

To avoid segmentation of the JS community (as we could probably have with Vuex and Pinia, or the Option API and Composition API), we should find a way to merge our effort to provide one solution, or two solutions which work great together.

What do you think ?

**Comment by antfu** [comment]
We are already working together. Peeky's Vite support is basically from Vitest, while we @Akryum also provides great ideas on improvements like including `happy-dom` and improving externalization detection. I think in the short term Peeky and Vitest will develop own each own while exchanging techs and ideas, which would be a good way for new ideas to be born IMO. In the long term, we will think about finding some ways to join the force I believe.

**Reply by edimitchel** [reply]
I discuss with Guillaume, I'm not telling you're not doing it already. I just want to make sure this is officially followed and communicated to all :)
